libc/test/include/header-test-template.c

```diff
@@ -0,0 +1,14 @@
+/*===-- Test for <@HEADER_NAME@> ----------------------------------------===
+ *
+ * Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDSList-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
```suggestion
 * SPDXList-License-Identifier: Apache-2.0 WITH LLVM-exception
```
(for a follow up cleanup); dunno how that snuck in, but it occurs in other places. Don't know how I even noticed that...

Reply:
I copied it from some other random lilbc file (and then had to change the `//` comments to be C89 compatible).  But I'm pretty sure I didn't introduce the typo, so we should grep for that typo in other files.

Reply:
Most or all other `libc/test/include` files have this typo.

Reply:
https://github.com/llvm/llvm-project/pull/122776